### PR TITLE
Add guide to copy file source connector to container

### PR DESCRIPTION
**Pull request by Haaroon**
Added instruction to copy file source connector into docker image. 
Adjusted numbers accordingly

### Motivation

* I am testing out the file source connector and it would not run without this command

### Modifications

I have adjusted the guide to tell the user to copy the connector to the docker instance

### Verifying this change

This change is already covered by existing tests

### Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency):  no
  - The public API:  no
  - The schema:  no 
  - The default values of configurations:  no
  - The wire protocol:  no
  - The rest endpoints:  no
  - The admin cli options: no
  - Anything that affects deployment: no 

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [x] `doc`

### CODE REVIEW

File: site2/docs/io-file-source.md [hunk 1]
```diff
@@ -105,7 +105,13 @@ Here is an example of using the File source connecter.
...
     ```
 
-6. Start the File source connector.
+6. Copy the File source connector to the container. 
+
+    ```bash
+    $ docker cp pulsar-io-file-{version}.nar pulsar-standalone:/pulsar/
+    ```
+
+7. Start the File source connector.
```
Comment (Anonymitaet) [inline site2/docs/io-file-source.md:114]: @tuteng can you help this from a technical perspective? Thanks

File: site2/docs/io-file-source.md [hunk 2]
```diff
@@ -105,7 +105,13 @@ Here is an example of using the File source connecter.
     $ curl -O https://mirrors.tuna.tsinghua.edu.cn/apache/pulsar/pulsar-{version}/connectors/pulsar-io-file-{version}.nar
     ```
 
-6. Start the File source connector.
+6. Copy the File source connector to the container. 
+
+    ```bash
+    $ docker cp pulsar-io-file-{version}.nar pulsar-standalone:/pulsar/
```
Comment (tuteng) [inline site2/docs/io-file-source.md:111]: Copy it to the `connectors` folder, then restart the container

```
docker cp pulsar-io-file-{version}.nar pulsar-standalone:/pulsar/connectors/
docker restart pulsar-standalone
```

Comment (Anonymitaet) [reply site2/docs/io-file-source.md:111]: Hi @Haaroon any feedback?

**Comment by Haaroon** [comment]
Perfect, i have added the suggestions, thank you

File: site2/docs/io-file-source.md (hunk 2 above)
Comment (Anonymitaet) [reply site2/docs/io-file-source.md]: Hi @tuteng, @Haaroon has incorporated your comment, any other technical comments?

File: site2/docs/io-file-source.md [hunk 3]
```diff
@@ -105,25 +105,32 @@ Here is an example of using the File source connecter.
     $ curl -O https://mirrors.tuna.tsinghua.edu.cn/apache/pulsar/pulsar-{version}/connectors/pulsar-io-file-{version}.nar
     ```
 
-6. Start the File source connector.
+6. Copy it to the `connectors` folder, then restart the container
```
Comment (Anonymitaet) [inline site2/docs/io-file-source.md:108]:
```suggestion
6. Copy it to the `connectors` folder, then restart the container.
```

**Review by Anonymitaet — APPROVED** [review]

**Comment by Haaroon** [comment]
all requested changes have been commited.

**Review by tuteng — APPROVED** [review]

**Pull request merged**